docs/src/testing/testing.md

```diff
@@ -91,3 +91,27 @@ Running 1 test(s) from src/
 [PASS] src::should_panic_check_data
 Tests: 1 passed, 0 failed, 0 skipped
 ```
+
+## Ignoring Some Tests Unless Specifically Requested
+
+Some tests can be very time-consuming to execute, so you might want to exclude them during most run of `snforge test`.
+You can achieve it using `#[ignore]` - tests marked with this attribute will not be run by default.
+
+```rust
+#[test]
+#[ignore]
+fn expensive_test() {
+    // perform very time-consuming operations
+}
+```
+
+```shell
+$ snforge
+Collected 1 test(s) from package_name package
+Running 1 test(s) from src/
+[IGNORE] src::expensive_test
+Tests: 1 passed, 0 failed, 0 skipped
+```
+
+You can run only the ignored tests using `snforge --ignored`. 
+To run all tests regardless of the `#[ignore]` attribute use `snforge --include-ignored`.
```

Comment:
Perhaps an example output with `--include-ignored` or `--ignored` should be added here? Though I'm not sure.

Reply:
I don't think it is that important to clog the docs with it